packages/react/src/filter-sidebar/docs/overview.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Filter sidebar
+description: Used for filtering options in catalogue pages.
```

Comment:
I think the current component description could be improved. Maybe something like this?

```suggestion
description: Filter sidebar is used for displaying filtering options in a search result or catalogue page.
```